openedx/core/djangoapps/user_api/views.py

```diff
@@ -240,6 +271,23 @@ def get(self, request):
 
         return HttpResponse(form_desc.to_json(), content_type="application/json")
 
+    @staticmethod
+    def field_options(form, field_name):
```

Comment:
I don't think this is worthy of being a method, since it's only really used in one place. Why not just put

```python
field_options = getattr(getattr(custom_form, 'Meta', None), 'serialization_options', {})
```

on the line preceding your call to `form_desc.add_field(...)` ?